randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1027,6 +1035,7 @@ Extra - room_id: [38]
   > Pickup (Energy Tank)
       All of the following:
           Speed Booster
+          # Wave beam could potentially make the room harder/impossible to complete
```

Comment:
We will likely have a `disable this item` system similar to Super so shouldn't be an issue. Noted though.